src/main/java/qna/domain/Question.java

```diff
@@ -84,12 +88,24 @@ public boolean isDeleted() {
         return deleted;
     }
 
-    public List<Answer> getAnswers() {
+    public Answers getAnswers() {
         return answers;
     }
 
     @Override
     public String toString() {
         return "Question [id=" + getId() + ", title=" + title + ", contents=" + contents + ", writer=" + writer + "]";
     }
+
+    public List<DeleteHistory> delete(User loginUser) throws CannotDeleteException {
+        if (!isOwner(loginUser)) {
+            throw new CannotDeleteException("질문을 삭제할 권한이 없습니다.");
+        }
+        deleted = true;
+
+        List<DeleteHistory> deleteHistorys = new ArrayList<>();
+        deleteHistorys.add(DeleteHistory.of(ContentType.QUESTION, getId(), writer, LocalDateTime.now()));
+        deleteHistorys.addAll(answers.delete(loginUser));
+        return deleteHistorys;
```

Comment:
```suggestion
        List<DeleteHistory> deleteHistories = new ArrayList<>();

        deleteHistories.add(deleteQuestion(loginUser));
        deleteHistories.addAll(answers.delete(loginUser));

        return deleteHistories;
```

이런 형태로 코드가 동작 할 수 있게 해보면 좋을 것 같아요. :)